openquake/commonlib/writers.py

```diff
@@ -121,6 +121,7 @@ def write(self, descrs):
     '(slight):float32',
     '(moderate):float32',
     '(extensive):float32',
+    '(extreme):float32',
```

Comment:
I know. This is just to fix the headers in the .csv files in our tests and demos. Otherwise the headers will be littered with `:float32` postfixes. Solving the issue in general is outside the scope of this PR.